docs/operators/jsonParser.md

```diff
@@ -0,0 +1,31 @@
+Json Parser
+=============
+
+## Operator Objective
+Purpose of this operator is to parse JSON records and construct a Plain Old Java Object ("POJO") out of it. The operator also emits each record as JSONObject. User can also provide a schema describing JSON data to validate incoming JSON records. Valid records will be emitted as POJO / JSONOnject while invalid ones are emitted on error port with error message.
+
+Json Parser does not hold any state and is **idempotent**, **fault-tolerance** & **statically/dynamically partitionable**.
```

Comment:
The claim about "statically/dynamically partitionable" needs supporting information showing the user how to do that partitioning. 

The claim that it does not hold any state is incorrect: `jsonSchema` and `parsedOutputCount` are part of the state; the base class also has additional state. Suggest omitting that claim.